datalad/tests/test_classes.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
```test_classes``` is an unfortunate name.   It is common to have a test_<filename>.py for   a <filename>.py  file you are testing.  So split into ```test_gitrepo.py```, etc